### PR TITLE
Tag RecurrenceAnalysis v0.1.0 [https://github.com/heliosdrm/Recurrenc…

### PR DESCRIPTION
**Pull request by heliosdrm**
…eAnalysis.jl]

### CODE REVIEW

File: RecurrenceAnalysis/versions/0.1.0/requires [hunk 1]
```diff
@@ -0,0 +1,3 @@
+julia 0.4
+Distances
+Compat
```
Comment (tkelman) [inline RecurrenceAnalysis/versions/0.1.0/requires]: best to declare a minimum version dependency that implements the features you need

Comment (heliosdrm) [inline RecurrenceAnalysis/versions/0.1.0/requires]: Ok, done in a separate pull request
